content/native_setup/android.md

```diff
@@ -0,0 +1,181 @@
+# Installing SCION on an Android device
+
+## Introduction
+It is possible to run SCION on an Android device. For this purpose the [Termux](https://github.com/termux/termux-app) app is used, which emulates a Terminal environment with the Linux base system that Android is based upon.
+
+The Termux app can be downloaded [on the Google Play Store](https://play.google.com/store/apps/details?id=com.termux).
+
+## Prerequisites
+
+For the course of this tutorial advanced use of Termux is required. Make yourself familiar with it by reading on the [Wiki](https://wiki.termux.com/wiki/Main_Page) about how the app can be used more comfortably.
+
+The tutorial also expects basic knowledge about how to interact with an Android device through the Android Device Bridge (ADB).
+
+### Install necessary packages
+
+Install the required packages from within Termux:
+
+```shell
+pkg install -y termux-setup-storage termux-exec ssh git python python2 clang make python-dev libffi-dev openssl-dev openssl-tools
+```
+
+### Configure Go workspace
+
+SCION requires a specific Go version. Termux' Go package is usually ahead of the required version. You can build the required packages yourself (e.g. using [termux-packages](https://github.com/termux/termux-packages)) or use the prebuilt package from the SCION repository:
```

Comment:
Suggestion for minor improvement is to add the text in brackets:
[The] Termux' Go package ...
e.g.[,]